apps/web/app/environments/[environmentId]/settings/members/EditMemberships.tsx

```diff
@@ -18,11 +33,104 @@ import toast from "react-hot-toast";
 import AddMemberModal from "./AddMemberModal";
 import CreateTeamModal from "@/components/team/CreateTeamModal";
 import { capitalizeFirstLetter } from "@/lib/utils";
+import { useProfile } from "@/lib/profile";
+import { EditIcon } from "lucide-react";
 
 type EditMembershipsProps = {
   environmentId: string;
 };
 
+type Role = {
```

Comment:
to be consistent with other frontend files please use
```
interface RoleElementProps {
```
as name for typed props 😊